php/class-post-type.php

```diff
@@ -522,11 +491,6 @@ public function save( array $args ) {
 			),
 		);
 		if ( ! empty( $args['status'] ) ) {
-			if ( isset( $args['post_date'], $args['edit_date'], $args['post_date_gmt'] ) ) {
-				$post_arr['post_date'] = $args['post_date'];
-				$post_arr['edit_date'] = $args['edit_date'];
-				$post_arr['post_date_gmt'] = $args['post_date_gmt'];
-			}
```

Comment:
Shouldn't the `save` method here be calling `\WP_Customize_Manager::save_changeset_post()`?